test/TalisOrm/AggregateRepositoryTest/Line.php

```diff
@@ -101,6 +106,8 @@ public function quantity()
 
     public function state()
     {
+        $this->isNew = false;
```

Comment:
this has a side effect: if you call this method twice (let's say a logger middleware or something), then at the time the entity arrives at the repository, it would be seen as an update, right?

Reply:
See also #24!